stack/hls_constructs/aws_batch_infra.py

```diff
@@ -0,0 +1,13 @@
+from constructs import Construct
+
+
+class BatchInfra(Construct):
+    """AWS Batch compute environment and queues."""
+    def __init__(
+        self,
+        scope: Construct,
+        id: str,
```

Comment:
`id` is a builtin, so we should avoid shadowing it by using the name `id_` instead.

Reply:
+1 I like the `construct_id` I've seen you use over the `id` that AWS examples use 🚀